frontend/src/components/select/index.tsx

```diff
@@ -0,0 +1,85 @@
+/** @jsx jsx */
+import React, { CSSProperties } from "react";
+import { default as ReactSelect, Props } from "react-select";
+import { jsx } from "theme-ui";
+
+export const Select: React.SFC<Props> = props => (
+  <ReactSelect
+    styles={{
+      control: (base: CSSProperties) => ({
+        ...base,
+        position: "relative",
+        border: "4px solid #000",
+        background: "#fff",
+        padding: "20px 8px",
+        overflow: "visible",
+
+        "> div": {
+          overflow: "visible",
+        },
+
+        "&:hover": {
+          borderColor: "#000",
+        },
+      }),
+      menu: (base: CSSProperties): CSSProperties => ({
+        ...base,
+        border: "4px solid #000",
+      }),
+      option: (base: CSSProperties, state: { isSelected: boolean }) => ({
+        ...base,
+        fontFamily: "aktiv-grotesk-extended,sans-serif",
+        background: state.isSelected ? "#f6f6f6" : "#fff",
+        "&:hover, &:focus": {
+          background: "#f6f6f6",
+        },
+      }),
+      container: (base: CSSProperties): CSSProperties => ({
+        ...base,
+        overflow: "visible",
+      }),
+      dropdownIndicator: (base: CSSProperties): CSSProperties => ({
+        ...base,
+        color: "#000",
+      }),
+      clearIndicator: (base: CSSProperties): CSSProperties => ({
+        ...base,
+        color: "#000",
+      }),
+      multiValueRemove: (
+        base: CSSProperties,
+        state: { isFocused: boolean },
+      ) => ({
+        ...base,
+        position: "absolute",
+        top: "-10px",
+        right: "-10px",
+        width: "26px",
+        height: "26px",
+        border: "3px solid #000",
+        background: state.isFocused ? "#FFBDAD" : "orange",
+        borderRadius: "100%",
+        cursor: "pointer",
+        color: state.isFocused ? "#DE350B" : "#000",
+      }),
+      placeholder: (base: CSSProperties): CSSProperties => ({
+        ...base,
+        fontFamily: "aktiv-grotesk-extended,sans-serif",
```

Comment:
we could inherit this :)

Reply:
makes sense, I want to make `react-select` inherit our theme so we do not have to do this (I think)